projects/platforms/index.html

```diff
@@ -1,17 +1,35 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width,initial-scale=1">
-    <title>Timeline</title>
-    <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=Nunito&display=swap" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css2?family=Zilla+Slab:wght@500&display=swap" rel="stylesheet">
-    <link href="./style.css" rel="stylesheet">
-    <script async defer src="./bundle.js"></script>
-</head>
-
-<body></body>
-
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width,initial-scale=1" />
+    <title>
+      US Election 2020 Timeline: Frequency and Timing of Platform Policy Changes
+    </title>
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link rel="icon shortcut" href="favicons/favicon.ico" />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="favicons/favicon-16x16.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="favicons/favicon-32x32.png"
+    />
```

Comment:
as only-ever-for-embedding content, favicon code and files are not needed, let's keep them out of the s3 bucket.

Reply:
turns out we want to send folks to the original page as  "larger version" experience, so favicons make sense.